packages/docs/src/components/layout/VersionSwitcher.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { ChangeEvent } from 'react';
+import corePackage from '../../../../design-system/package.json';
+import healthcarePackage from '../../../../ds-healthcare-gov/package.json';
+import medicarePackage from '../../../../ds-medicare-gov/package.json';
+import themes from '../../../../../themes.json';
+import versions from '../../../../../versions.json';
+import useTheme from '../../helpers/useTheme';
+import { Dropdown } from '@cmsgov/design-system';
+
+function getVersionOptions(versions: string[]) {
+  return versions.map((version) => ({
+    label: version,
+    value: version,
+  }));
+}
+
+function getPackageData(theme: string) {
```

Comment:
Wish we could do this without hard coding our packages and using themes.json instead.  Maybe with an additional field in themes.json for the package.json path, then a dynamic import of the json for that theme?

Reply:
When I was writing this, I was thinking that I'd be able to do it with Astro if I could keep the part that feeds this component completely server-side. It's not so straightforward to do this with Gatsby. In fact, I'm not sure how to do it without adding it to each page's _context_ through `packages/docs/gatsby-node.js` or creating a plugin that makes that data available through GraphQL 😵

Reply:
If you have ideas, I'd love to remove the hardcoding!

Reply:
Oof yeah I was hoping it would be as easy as I mentioned, that's too bad. I mean we have these kinds of references in several places, probably just best to do a full CMS integration and update the code around the codebase where needed when that happens. It's not like we add child systems too often.